.kitchen.yml

```diff
@@ -394,6 +394,19 @@ suites:
               my_consumer:
                 my_topic: [0, 1, 4, 12]
 
+- name: datadog_kubernetes
```

Comment:
actually could you remove the integration test and the related dir `test/integration/datadog_kubernetes`?

These tests are deprecated for integration recipes, we're just keeping the tests there until we move them all to spec tests.

Once you've removed them I'll merge, thanks 👍